src/System.Linq.Expressions/tests/Call/CallTests.cs

```diff
@@ -82,6 +82,19 @@ public static void ArrayWriteBack(bool useInterpreter)
             Assert.Equal(2, lambda(array));
         }
 
+        [Theory]
+        [ClassData(typeof(CompilationTypes))]
+        public static void EnumArgAndReturn(bool useInterpreter)
+        {
+            ParameterExpression p = Expression.Parameter(typeof(NonGenericClass.E1));
+            MethodCallExpression call = Expression.Call(typeof(NonGenericClass).GetMethod("FooEnum"), p);
+            Func<NonGenericClass.E1, NonGenericClass.E2> lambda = Expression.Lambda<Func<NonGenericClass.E1, NonGenericClass.E2>>(call, p).Compile(useInterpreter);
+
+            object boxed = new Mutable();
+            Assert.Equal(NonGenericClass.E2.One, lambda(NonGenericClass.E1.One));
```

Comment:
how is this being used?

Reply:
'boxed" ?    It is probably a leftover from when I tried a different implementation for the test. I will remove it.

---
In reply to: [181906624](https://github.com/dotnet/corefx/pull/28792#discussion_r181906624) [](ancestors = 181906624)